projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -172,6 +176,10 @@ describe('AuthService', () => {
             provide: Location,
             useValue: locationSpy,
           },
+          {
+            provide: AuthClientConfig,
+            useValue: authClientConfigSpy,
+          },
```

Comment:
Thinking about our other conversations we've had about mocking less, do we need to use a Spy here or can we just give it an an instance of `AuthClientConfig` with the right config?

Reply:
As the config is different between tests, mocking it makes it a bit easier I think.